client-js/README.md

```diff
@@ -0,0 +1,170 @@
+# CEP-78 JavaScript Client Tutorial
+
+This tutorial outlines usage of the JavaScript client available for the CEP-78 Enhanced NFT Standard.
+
+Further information on the CEP-78 Enhanced NFT Standard itself can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+## Requirements
+
+The JavaScript client for CEP-78 requires the use of the CasperLabs SDK for JavaScript. Information on installation and use of the Casper JS SDK is available [here](https://github.com/casper-ecosystem/casper-js-sdk).
```

Comment:
The client is a standalone library that will be published through the npm just like our previous clients, eg. https://www.npmjs.com/package/casper-cep47-js-client

Reply:
So it won't require the js-sdk :)